package.json

```diff
@@ -58,30 +55,28 @@
     "@wdio/spec-reporter": "5.23.0",
     "@wdio/sync": "5.23.0",
     "chai": "4.2.0",
-    "clean-css-cli": "4.3.0",
     "coveralls": "3.1.0",
     "eslint": "7.12.1",
     "eslint-plugin-node": "6.0.1",
-    "faucet": "0.0.1",
-    "htmllint-cli": "0.0.7",
     "jest": "26.4.2",
-    "jest-tap-reporter": "1.9.0",
     "node-mocks-http": "1.9.0",
     "nodemon": "2.0.4",
     "npm-audit-ci-wrapper": "3.0.1",
     "onchange": "6.1.1",
-    "postcss-cli": "^8.3.1",
     "redis-mock": "0.52.0",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "18.3.0",
     "stylelint-scss": "^3.19.0",
-    "uglify-js": "3.11.5",
     "wdio-docker-service": "2.4.0",
-    "wdio-image-comparison-service": "1.13.6",
+    "wdio-image-comparison-service": "^3.0.0",
     "wdio-video-reporter": "2.0.1"
   },
+  "peerDependencies": {
+    "sass": "^1.42.1"
```

Comment:
Does Sass need to be a `dependency` _and_ a `peerDependency`?

Reply:
I think so... It's definitely a peer dep, so it earned a spot in `peerDependencies`. But, `peerDependencies` are not automatically installed with `npm install` as of this writing.  So by placing it in `dependencies` it gets installed automatically, including in our automated builds.  Removing it from `peerDependencies` triggers an eslint error, and removes a nice-to-have context on why that package is installed.  Open to suggestions!

Reply:
Weird, that's a real head scratcher. I don't think I'm reproducing the ESLint error locally when removing it as a peer dependency.

```sh
gh pr checkout 2316
nvm i 14.17
rm -rf node_modules
npm i
npm run lint
```

That succeeds, but if I remove the peerDependency and then delete the ./node_modules/ (and package-lock.json) and reinstall everything, it still works (macOS 11).

I think I was just confused because I didn't see any mentions of needing peer dependencies in https://github.com/dlmanning/gulp-sass docs.
But hey, if it works, probably best to leave it alone.

Reply:
Looks like it was an error triggered by an eslint plugin, `eslint-plugin-node`.  This is the error I have in my history, tho admittedly I can't reproduce it now when removing `peerDependencies`:
https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-require.md

I thought it was a head-scratcher indeed, but after adding `Sass` to peerDeps, that error went away from me.  Maybe it was a coincidence.

Regarding mentions in the gulp-sass docs, you're right – it's not specifically listed as a **_peer_** dependency, but requiring a 3rd party compiler is specifically mentioned in the _installation_ section: https://github.com/dlmanning/gulp-sass#installation

I took this to suggest a peer dependency, since it's not required by our repo.  

Reading into it further, it seems the right solution is to add Sass to the `peerDependency` section and not include it in `dependencies`.  For us, I don't think this will work unless we update our automated builds and add documentation specifying that a manual install is required.  From [the docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependencies), peer dependencies are not auto-installed for our version of npm (6).  They are for version 7, which we should support next time we bump our NodeJS version.

Reply:
Awesome, thanks for the explanation. One of these days I'll sit down and actually understand what that error means. It's managed to elude me for years and I usually end up just disabling it.
Plus I'm always confused by Node's stance on "peerDeps are bad. No wait, now peerDeps are OK again", but nice to see them coming back in npm@7.